tools/tool-env/profile-require.ts

```diff
@@ -0,0 +1,185 @@
+// Use path instead of files.ts here because we are explicitly trying to
+// track requires, and files.ts is often a culprit of slow requires.
+import path from 'path';
```

Comment:
Since there seems to be ample usage of both kinds of string quote in the codebase, I was unsure about this and decided to just leave the string the way it was (i.e., as a single string).

Before:
```js
var path = require('path');
```

Actually, the JS style guide you linked below says it should be single quotes, although I do agree with you that there are a lot of double quotes in the codebase:
https://github.com/meteor/javascript/#strings

That said, I don't think this kind of thing affects readability of the code at all or is worth bikeshedding over even a little bit! I'm happy to adjust to whatever the Meteor team wants (preferably, they would just install Prettier 😅 ) but am trying to just leave things the way they are where it's not clear what they want.